services/web-server/config.yml

```diff
@@ -58,10 +58,8 @@ defaults:
     #   clientId: !env GITHUB_CLIENT_ID
     #   clientSecret: !env GITHUB_CLIENT_SECRET
     strategies: !env:json UI_LOGIN_STRATEGIES
-    jwt:
-      # this should be a random string, and is used as input to the HS256 JWT
-      # signing algorithm
-      key: !env JWT_KEY
+    # this should be a random string, and is used to create the session ID
+    sessionSecret: !env SESSION_SECRET
```

Comment:
This will need to be mentioned in the changelog, since deployers will need to update their config.